### PR TITLE
KIALI-565 Show the graph time range on screen

### PR DESCRIPTION
**Pull request by jshaughn**
Note the time range in the top right corner:

image

This UI PR requires Server PR: https://github.com/kiali/kiali/pull/750

### CODE REVIEW

**Comment by abonas** [comment]
this can be a nice addition! 
Was this discussed with UX?
Couple of questions:
1. I see that the timezone is not mentioned - how can the user know which TZ does that range relate to?
2. Does that range that's displayed autorefreshed?
3. Why is the range placed on top of the other controls and not next to them?

File: src/pages/Graph/GraphPage.tsx [hunk 1]
```diff
@@ -34,10 +34,11 @@ type GraphURLPathProps = {
 
 type ReduxProps = {
   activeNamespaces: Namespace[];
-  duration: DurationInSeconds;
+  duration: DurationInSeconds; // current duration (dropdown) setting
   edgeLabelMode: EdgeLabelMode;
   graphData: any;
-  graphTimestamp: number;
+  graphDuration: number; // duration used for current graph
```
[the remark this answers is not in the document]
Comment (jshaughn) [reply src/pages/Graph/GraphPage.tsx]: OK, I'll change the types.

**Comment by jshaughn** [comment]
@abonas This has been around since April and UX had not added any comments, so I figured it would be best to get something in place and adjust if necessary.

The time is presented in an abbreviated way because huge datetime strings are clunky and in this scenario, unnecessary.  The times are in the TZ and locale of the browser (so the user's perspective).  The range updates when the graph updates.  The proposed range placement takes advantage of currently unused space, and I put it above the range selection because the two elements are both temporal.

**Comment by abonas** [comment]
> @abonas This has been around since April and UX had not added any comments, so I figured it would be best to get something in place and adjust if necessary.

I don't see anyone from UX being asked/tagged on the JIRA, so they would not know their input is needed.
I only see @mwringe  comment regarding the *need* to open a question on the design repo before implementing anything,  was the issue opened?
https://issues.jboss.org/browse/KIALI-565?focusedCommentId=13669417&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13669417

> 
> The time is presented in an abbreviated way because huge datetime strings are clunky and in this scenario, unnecessary. The times are in the TZ and locale of the browser (so the user's perspective). The range updates when the graph updates. The proposed range placement takes advantage of currently unused space, and I put it above the range selection because the two elements are both temporal.

**Comment by jshaughn** [comment]
@abonas OK, When this is merged I'll open a UX design issue for any feedback they may have.

**Pull request merged**